osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-monitoring.xsd

```diff
@@ -409,131 +412,134 @@
 
-  <xsd:element name="ProfileGenericDataRequest">
+  <xsd:element name="GetPowerQualityProfileRequestData">
     <xsd:complexType>
       <xsd:complexContent>
-        <xsd:extension base="tns:ProfileGenericDataRequestData">
+        <xsd:extension base="tns:PowerQualityProfileRequestData">
           <xsd:sequence>
-            <xsd:element name="DeviceIdentification" type="common:DeviceIdentification" />
+            <xsd:element name="DeviceIdentification" type="common:DeviceIdentification"/>
           </xsd:sequence>
         </xsd:extension>
       </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
 
-  <xsd:complexType name="ProfileGenericDataRequestData">
+  <xsd:complexType name="PowerQualityProfileRequestData">
     <xsd:complexContent>
       <xsd:extension base="common:Query">
         <xsd:sequence>
-          <xsd:element name="ObisCode" type="common:ObisCodeValues" />
-          <xsd:element name="BeginDate" type="xsd:dateTime" />
-          <xsd:element name="EndDate" type="xsd:dateTime" />
-          <xsd:element name="SelectedValues" type="common:CaptureObjectDefinitions" minOccurs="0" />
+          <xsd:element name="ProfileType" type="xsd:string"/>
+          <xsd:element name="BeginDate" type="xsd:dateTime"/>
+          <xsd:element name="EndDate" type="xsd:dateTime"/>
+          <xsd:element name="SelectedValues" type="common:CaptureObjectDefinitions" minOccurs="0"/>
         </xsd:sequence>
       </xsd:extension>
     </xsd:complexContent>
   </xsd:complexType>
 
-  <xsd:element name="ProfileGenericDataAsyncResponse">
+  <xsd:element name="GetPowerQualityProfileAsyncResponse">
     <xsd:complexType>
       <xsd:complexContent>
         <xsd:extension base="common:AsyncResponse"></xsd:extension>
       </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
 
-  <xsd:element name="ProfileGenericDataAsyncRequest">
+  <xsd:element name="GetPowerQualityProfileAsyncRequest">
     <xsd:complexType>
       <xsd:complexContent>
         <xsd:extension base="common:AsyncRequest"></xsd:extension>
       </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
 
-  <xsd:element name="ProfileGenericDataResponse">
+<!--
+  <xsd:element name="GetPowerQualityProfileResponse">
     <xsd:complexType>
-        <xsd:complexContent>
-            <xsd:extension base="tns:ProfileGenericData"></xsd:extension>
-        </xsd:complexContent>
+      <xsd:complexContent>
+        <xsd:extension base="tns:GetPowerQualityProfileResponseData"></xsd:extension>
+      </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
+-->
 
-  <xsd:complexType name="ProfileGenericDataResponseData">
+  <xsd:complexType name="GetPowerQualityProfileResponse">
     <xsd:complexContent>
       <xsd:extension base="common:Response">
         <xsd:sequence>
-          <xsd:element name="ProfileGenericData" type="tns:ProfileGenericData" />
+          <xsd:element name="PowerQualityProfileDatas" type="tns:PowerQualityProfileData"
```

Comment:
this is change.

Reply:
Ik begrijpo dat 'GetProfileGenericData' niet langer beschikbaar is omdat deze is vervangen door 'GetPowerQualityProfile'?

Reply:
ja vervangen en aangepast. ProfileGenericData is obsolete.